standalone/ansible/requirements.yml

```diff
@@ -15,7 +15,7 @@ roles:
   - src: bdellegrazie.haproxy_exporter
   - src: bdellegrazie.nginx_exporter
   - src: datadog.datadog
-    version: 4.13.0
+    version: 4.15.0
```

Comment:
Could I gently suggest that we merge https://github.com/simpledotorg/deployment/pull/424 first, and move this change over to https://github.com/simpledotorg/simple-standalone?